test/live/test_blocking_hooks.rb

```diff
@@ -462,6 +462,33 @@ def task.configure
             end
         end
 
+        describe "stop transition timeout" do
+            attr_reader :deployment
+
+            before do
+                @task = syskit_deploy_configure_and_start(
+                    OroGen.orogen_syskit_tests.Empty
+                          .deployed_as("stop-transition-timeout")
+                )
+                @deployment = @task.execution_agent
+                @__stop_transition_timeout = Syskit.conf.stop_transition_timeout
+            end
+
+            after do
+                cleanup_running_tasks
+                Syskit.conf.stop_transition_timeout = @__stop_transition_timeout
+            end
+
+            it "detects a task that should have stopped but has not" do
+                Syskit.conf.stop_transition_timeout = 1
+                @task.should_receive(:stop_orocos_task)
+                tic = Time.now
+                expect_execution { @task.stop! }
+                    .to_quarantine(@task)
+                assert (Time.now - tic) > 1
```

Comment:
I think lately we are moving to 
```suggestion
                assert_operator Time.now - tic, :>, 1
```

but, to be clear, it is not a big deal for me and is not in the code guidelines whatsoever